### PR TITLE
Replace babel-polyfill with transform-runtime

### PR DESCRIPTION
**Pull request by ponelat**
For integrating this lib into other apps, that already include babel-polyfill

### CODE REVIEW

**Comment by shockey** [comment]
My concern here is IE11, `babel-polyfill` fixed a `Symbol` reference error for it in UI a couple months ago.

**Comment by ponelat** [comment]
Ok, so we'd need to test it in IE11 to know for sure.
The differences are in _instance_ methods, which are _NOT_ covered by transform-runtime. 

Babel-polyfill is a big blocker for using swagger-ui as a component. Since there can only be one reference. An alternative can be:  ignoring the call to babel-polyfill if one has already been called. 
Not sure if that functionality is going to land in babel-polyfill, I know its mentioned there.

So tl;dr... we should either ensure our the code works with transform-runtime. Or add a workaround for having two instances. ( Assuming we want swagger-ui as a component... :D )

**Comment by webron** [comment]
@ponelat @shockey ?

**Comment by ponelat** [comment]
If we can lets use transform-runtime.
It means stripping out any modern _instance_ methods. We can get a list and just prune away.  Or add those polyfills manually ( ie: one-by-one on an as-needed basis ).

The alternative is to patch babel-polyfill to run multiple times ( it should be idiomatic, but for some reason they decided to throw an error if there are multiple calls to it. I'm curious as to why ).

**Comment by ponelat** [comment]
An idea is to use https://github.com/amilajack/eslint-plugin-compat to _lint_ the code and detect instance methods that aren't supported. It _might_ work :D

**Comment by ponelat** [comment]
This now works in IE 11, so we're good for that review @shockey.
Keep in mind, that we can always fallback to using babel-polyfill in the standalone version - its the lib we're concerned with.

**Comment by ponelat** [comment]
Also need https://github.com/swagger-api/swagger-js/pull/1088 to work

**Comment by shockey** [comment]
LGTM for IE11:

image
